### ISSUE TITLE
[ts] Cannot find name 'invalidstring'.

### ISSUE THREAD

**Issue by charlzyx** [palantir/tslint#3202]
### Bug Report

- __TSLint version__: 5.7.0
- __TypeScript version__:2.5.2
- __Running TSLint via__:  CLI 

#### TypeScript code being linted

```ts
import React, {
  Component,
  // @types/react
  ReactChildren,
} from 'react';

ddd ; /*this line*/

```

with `tslint.json` configuration:

```json
{
	"extends": ["tslint-config-airbnb", "tslint-react"],
	"rules": {
		"import-name": [0]
	}
}
```

#### Actual behavior
![image](https://user-images.githubusercontent.com/18055018/30102534-879881ec-9322-11e7-883a-47a288efbb90.png)
It Seems that Code 's Linter find it , but when i use cli, tslint cannot find any error or warning;
![image](https://user-images.githubusercontent.com/18055018/30103824-66f3f7e2-9326-11e7-8938-893bbf0242fd.png)
Here is my package.json script
`"tslint": "node node_modules/tslint/lib/tslint-cli.js 'src/**/*.ts?(x)'",`
I have tried to output files and files matching;

#### Expected behavior
same with VS Code

**Comment by ajafff** [comment]
As you can tell from the `[ts]` prefix of the message, this error comes from TypeScript and not from tslint.
VSCode uses the TypeScript compiler behind the scenes to tell you what's wrong with your code.

TSLint is only responsible to show you hints/warnings about code smells, bad practices and so on.
From the usage docs https://palantir.github.io/tslint/usage/cli/

> Please ensure that the TypeScript source files compile correctly before running the linter.

If you really need to see this error when running tslint from CLI, you can enable type checking via `--type-check` flag: https://palantir.github.io/tslint/usage/type-checking/
